templates/commands/render/render.go

```diff
@@ -551,6 +602,8 @@ type stepParams struct {
 	scratchDir  string
 	stdout      io.Writer
 	templateDir string
+	// Temporary directory to hold debug information when debug is enabled.
+	debugDir string
```

Comment:
Rather than making this a field of stepParams, could we make it a parameter to executeSteps? My reasoning is that stepParams is provided to all the `action*` functions (like `actionInclude()` and `actionPrint()`) and I want to avoid providing any extraneous values to those functions that they're not supposed to touch.

Reply:
Will try your suggestion and see!